MovieMVP/ServiceLayer/NetworkService.swift

```diff
@@ -0,0 +1,121 @@
+// NetworkService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Alamofire
+import Foundation
+import SwiftyJSON
+
+/// Протокол сетевого сервиса
+protocol NetworkServiceProtocol {
+    // MARK: - Public Methods
+
+    func fetchMovies(categoryMovies: String, completion: @escaping (Result<[Movie]?, Error>) -> Void)
+    func setupImageFromURLImage(posterPath: String, completion: @escaping (Result<Data?, Error>) -> Void)
+    func fetchSimilarMovies(idMovie: Int, completion: @escaping (Result<Data?, Error>) -> Void)
+}
```

Comment:
```suggestion
protocol NetworkServiceProtocol {
    // MARK: - Public Methods

    func fetchMovies(categoryMovies: String, completion: @escaping (Result<[Movie]?, Error>) -> Void)
    func setupImageFromURLImage(posterPath: String, completion: @escaping (Result<Data?, Error>) -> Void)
    func fetchSimilarMovies(idMovie: Int, completion: @escaping (Result<Data?, Error>) -> Void)
}
```
все протоколы в отдельный файл